examples/custom-vpc-network/main.tf

```diff
@@ -73,6 +73,8 @@ module "lacework_gcp_agentless_scanning_project_multi_region_use1" {
   regional = true
 
   custom_vpc_subnet = google_compute_subnetwork.awls_subnet_1.id
+  # example: how to disable public IP address on scanning VMs in GCP
+  additional_environment_variables = [{name="USE_PUBLIC_IPS", value="false"}]
```

Comment:
Is this environment variable something your code is referencing; or is this a more general GCP-specific environment variable?

If the former, then I'd recommend having a Terraform module variable that lets people choose public or private IPs. That Terraform module variable can then be constructed into an environment variable here to be passed to the resources that Terraform creates.

We should avoid having people read documentation on what Terraform module variables to set, and what environment variables to set. Best practice is to stick to Terraform variables for whatever controls/functionality we introduce, but allow people to set environment variables for general purpose stuff like controlling HTTP clients with proxies or setting CSP-defined options.

Reply:
@theopolis 

This was more of an example which I can remove.

This functionality is needed because (a) it's supported by our AWS TF and (b) it's needed for things like HTTP_PROXY.

This environment variable, to your point, is internal and undocumented. I agree that the ideal solution for this is to change the TF provider itself to support it as a first class variable. This will take longer and is somewhat separate from allowing environment variables to pass. 

So I propose leaving this PR as is, with this example amended, and doing future work on the provider. OK?